src/pycram/datastructures/enums.py

```diff
@@ -120,3 +120,36 @@ class GripperType(Enum):
     CUSTOM = auto()
 
 
+class PerceptionTechniques(Enum):
+    """
+    Enum for techniques for perception tasks.
+    """
+    ALL = auto()
+    HUMAN = auto()
+    TYPES = auto()
+
+
+class ImageEnum(Enum):
```

Comment:
This can as well be IntEnum

Reply:
U mean renaming it? This enums are there for calling different images on the display on the hsr tho

Reply:
IntEnum is a class in enum that just indicates that the values can be casted to int and used as if they were ints. This is generally useful for enum conversion